_example/main.go

```diff
@@ -22,7 +22,7 @@ import (
 	log "github.com/go-pkgz/lgr"
 	"github.com/go-pkgz/rest"
 	"github.com/go-pkgz/rest/logger"
-	"github.com/golang-jwt/jwt"
+	oldjwt "github.com/golang-jwt/jwt"
```

Comment:
Is there any reason to keep old version of JWT for an example instead of updating it here as well?

Reply:
The external library wants to consume that specific function. Once the external library also upgrades, it will make sense to bump.

Reply:
Can you please clarify this part? What exactly "the external library" do you mean?

Reply:
oldjwt is used in only one place -- as input to `github.com/go-oauth2/oauth2/v4/manage`. Sure, you could rewrite it, but that's not what the `manage` package expects.

Reply:
I've [tried](https://github.com/go-pkgz/auth/compare/paskal/example_jwt_v5?expand=1), and I think it doesn't make sense. The dependency will move from direct to indirect, and example code will become more complicated. See the link for the results.